docs/usage/golang.md

```diff
@@ -38,5 +38,7 @@ Renovate will commit all files changed within the `vendor/` folder.
 
 #### Go binary version
 
-Currently, Renovate will try to keep up with the very latest version of `go`, and it is not configurable.
-It is planned though to support a configurable version of `go` per-repository soon.
+By default, Renovate will keep up with the very latest version of `go`.
+
+You can "pin" the `go` version that Renovate uses.
+Say you want Renovate to use Go version 1.14.2, you can do this by adding `go 1.14.2` to your `go.mod` file.
```

Comment:
@viceice What do you think of this?

Reply:
Sounds good. I think the version  is always present for modules since go 1.12.
https://golang.org/doc/go1.12#modules

I'm not sure if `go 1.14.2` will work, as I'm no go developper 🤔

Reply:
I've updated this to `1.14.12` which is one of the 2 stable versions listed on the Go download website: https://golang.org/dl/

Reply:
Pinning go patch version isn't supported, we would get `go: errors parsing go.mod:` error, so only `go 1.14` is allowed.

Reply:
I'll go edit the docs. :smile: